server/routes/index.ts

```diff
@@ -72,10 +58,32 @@ export default function routes(service: Services): Router {
     }
 
     const report = await incidentReportingApi.getReportWithDetailsById(id)
+    const reportedBy = (await manageUsersApiClient.getNamedUser(systemToken, report.reportedBy))?.name
     const prisonerNumbers = report.prisonersInvolved.map(pi => pi.prisonerNumber)
     const prisonersLookup = await offenderSearchApi.getPrisoners(prisonerNumbers)
 
-    res.render('pages/report', { report, prisonersLookup })
+    res.render('pages/report', { report, prisonersLookup, reportedBy })
+  })
+
+  get('/incidents', async (req, res, next) => {
+    const { user } = res.locals
```

Comment:
we ought to still prevent anyone from reaching this page in production with this from the deleted route

```javascript
    // No authorisation at this point, no data shown in production
    if (config.environment === 'prod') {
      response.content = []
    }
```

Reply:
Addressed in line 70.